core_lib/interface/recentfilemenu.cpp

```diff
@@ -31,6 +31,12 @@ QMenu( title, parent )
     mClearAction = new QAction( tr("Clear"), this ); // share the same translation
 }
 
+RecentFileMenu::~RecentFileMenu()
+{
+    delete mClearSeparator;
+    delete mClearAction;
+}
+
```

Comment:
It's not leaking actually. A QMenu will delete all owned QActions on destroying. Sort of Qt's memory management convention. But it is ok to do it explicitly though.